src/query-builder/DeleteQueryBuilder.ts

```diff
@@ -144,16 +146,35 @@ export class DeleteQueryBuilder<Entity> extends QueryBuilder<Entity> {
         return this;
     }
 
+    /**
+     * Optional returning/output clause.
+     */
+    returning(returning: string): this {
+        this.expressionMap.returning = returning;
+        return this;
+    }
+
+    output(output: string): this {
+         return this.returning(output);
+    }
+    
     // -------------------------------------------------------------------------
     // Protected Methods
     // -------------------------------------------------------------------------
 
     /**
-     * Creates DELETE express used to perform insert query.
+     * Creates DELETE express used to perform query.
      */
     protected createDeleteExpression() {
         const tableName = this.escape(this.getMainTableName());
-        return `DELETE FROM ${tableName}`; // todo: how do we replace aliases in where to nothing?
+        if (this.expressionMap.returning !== "" && this.connection.driver instanceof PostgresDriver) {
+            return `DELETE FROM ${tableName} RETURNING ${this.expressionMap.returning}`;
+
+        } else if (this.expressionMap.returning !== "" && this.connection.driver instanceof SqlServerDriver) {
+            return `DELETE FROM ${tableName} OUTPUT ${this.expressionMap.returning}`;
+        } else {
+            return `DELETE FROM ${tableName}`; // todo: how do we replace aliases in where to nothing?
```

Comment:
What does this todo mean? Aren't they disabled when "disableEscaping" is set to true?